### PR TITLE
Sets the threshold to 0 for necro health.

### PR DESCRIPTION
**Pull request by jayRyan24**
Sets the threshold to 0 for necro health to clean even if not needed by settings.

### CODE REVIEW

File: safe-room.lic [hunk 1]
```diff
@@ -14,10 +14,15 @@ class SafeRoom
 
   def initialize
     arg_definitions = [
-      [{ name: 'force', regex: /force/i, description: 'Run healing logic even if there are no visible wounds.' }],
-      [{ name: 'skip', regex: /(s|skip)/i, description: 'Skip running to the safe room, useful for Empaths and Necros.' }],
+    arg_definitions = [
+      [ { name: 'force', regex: /force/i, description: 'Run healing logic even if there are no visible wounds.' } ],
```
Comment (rpherbig) [inline safe-room.lic]: Would it be easier to just set `force` to `optional: true` instead of adding two more arg sets?

File: safe-room.lic [hunk 2]
```diff
@@ -14,10 +14,15 @@ class SafeRoom
 
   def initialize
     arg_definitions = [
```
Comment (rpherbig) [inline safe-room.lic]: Are there now two `arg_definitions = [` lines?

File: safe-room.lic (hunk 1 above)
Comment (jayRyan24) [reply safe-room.lic]: I thought if we did this the force would require the skip first?

File: safe-room.lic (hunk 2 above)
Comment (jayRyan24) [reply safe-room.lic]: It's a copy and paste error.

**Pull request merged**